app/controllers/user_journeys_controller.rb

```diff
@@ -6,4 +6,10 @@ def restart
 
     redirect_to user_journey.restart!.journey
   end
+
+  def in_process
```

Comment:
Mozno by sme to mohli dat rovno pod  `index` aby bolo jasne ze to je prehlad. cc @michal-rohacek ?